tests/performance/readme.md

```diff
@@ -1,56 +0,0 @@
-# Running performance tests
```

Comment:
Did you move the documentation too?

Reply:
@gyorb I removed this file because I think this readme is not relevant for us. The script was refactored fully.

Reply:
Ok, I thought we still use this documentation